deploy/charts/rook-ceph/values.yaml

```diff
@@ -636,6 +636,8 @@ hostpathRequiresPrivileged: false
 # -- Disable automatic orchestration when new devices are discovered.
 disableDeviceHotplug: false
 
+# -- The revision history limit for all pods created by Rook. If blank, the K8s default is 10.
+revisionHistoryLimit: 
```

Comment:
For consistency with other settings, please add a blank line after

Reply:
done